src/core_modules/capture-core/components/TopBarActions/TopBarActions.component.js

```diff
@@ -104,12 +99,12 @@ const ActionButtonsPlain = ({
                                     programName,
                                     interpolation: { escapeValue: false },
                                 })}
-                                onClick={() => { setOpenSearch(prev => !prev); onFindClick(); }}
+                                onClick={() => { onFindClick(); setOpenSearch(prev => !prev); }} // Rekkefølge endret
```

Comment:
Hey @henrikmv 
If the comment is necessary, please change it from Norwegian to English. There is also one on line 107. Apart from these, the code looks good 👏 
Thanks!

Reply:
@simonadomnisoru, they weren't supposed to be included, so I've removed them now. Thanks!